.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
I think this should be written in yaml, like the data and documentation templates?